src/beamlime/core/application.py

```diff
@@ -0,0 +1,79 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
+
+import asyncio
+from abc import ABC, abstractmethod, abstractstaticmethod
+from queue import Empty
+from typing import Union
+
+from colorama import Style
+from colorama.ansi import AnsiBack, AnsiFore, AnsiStyle
+
+
+class BeamlimeApplicationInterface(ABC):
+    _input_ch = None
+    _output_ch = None
+    verbose = None
+    verbose_option = None
+
+    def __init__(
+        self,
+        config: dict,
+        verbose: bool = False,
+        verbose_option: Union[AnsiFore, AnsiStyle, AnsiBack, str] = Style.RESET_ALL,
+    ) -> None:
+        self.parse_config(config)
+        self.verbose = verbose
+        self.verbose_option = verbose_option
```

Comment:
Why do application interfaces need to know about output styles?

Reply:
It was a temporary solution until I implement `logging`...! I will remove this as soon as we have proper loggers.

Reply:
I opened the issue #34 so it'll be resolved there.